spinn_utilities/make_tools/file_convertor.py

```diff
@@ -0,0 +1,453 @@
+import os
+import re
+import sys
+
+TOKEN = chr(30)  # Record Sperator
```

Comment:
Sperator vs separator

Reply:
chr(40) is the ( symbol. not a record separator/TOKEN. not sure if this is a good name for what your looking for

Reply:
30 != 40

Reply:
Technically, the correct separator would be the Unit Separator, `chr(31)`, as that was intended to be the separator between fields (why “unit”? I don't know…) but practically it could be anything unlikely to otherwise occur in normal output so it isn't worth changing.